dev/com.ibm.websphere.appserver.features/visibility/public/jaxws-2.2/com.ibm.websphere.appserver.jaxws-2.2.feature

```diff
@@ -30,6 +31,7 @@ IBM-ShortName: jaxws-2.2
 IBM-SPI-Package: com.ibm.wsspi.webservices.handler
 Subsystem-Name: Java Web Services 2.2
 -features=\
+ com.ibm.websphere.appserver.internal.optional.jaxws-2.2, \
```

Comment:
You added the new feature, but didn't remove the bundle that imports bundle as this feature. Was there a reason for having it listed "twice"? Maybe so that you can override the fact that the internal optional feature uses the property `require-java:="9"; ` on its bundle import statement? If that's the case then this is fine but we want to be sure that the public jaxws-2.2 feature never uses the jax-ws 2.2 spec from the JDK.

Reply:
We this statement allows us to handle the "tolerates" resolution between different jaxws features that use both features. We override the actual api's down below to make sure we always get the apis from liberty in this feature.